src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -95,7 +95,14 @@ module sonic-portchannel {
                     mandatory true;
                     type stypes:admin_status;
                 }
-            } /* end of list PORTCHANNEL_LIST */
+
+				leaf lacp_key {
+					type string {
+						/* lacp key should be either auto or a integer in the range of 1 to 65535 */
+						pattern '[1-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5]|auto';
```

Comment:
I find this expression is wrong ?  Shouldn't the starting regex be [1-9][0-9]{1,3}. You could use this link to check out https://regexr.com/